kaggle-classification/keras_trainer/rnn.py

```diff
@@ -24,15 +24,23 @@ def __init__(self, embeddings_matrix, hparams):
     self.hparams = hparams
 
   def get_model(self):
-    I = Input(shape=(self.hparams.sequence_length,), dtype='float32')
+    sequence_length = self.hparams.sequence_length
+
+    I = Input(shape=(sequence_length,), dtype='float32')
     E = Embedding(
         self.hparams.vocab_size,
         self.hparams.embedding_dim,
         weights=[self.embeddings_matrix],
         input_length=self.hparams.sequence_length,
```

Comment:
Nit: For consistency, I suppose this should be sequence_length instead of self.hparams.sequence_length.